docs/introduction/overview.mdx

```diff
@@ -153,356 +20,114 @@ description: "🧙 A modern replacement for Airflow."
-
-Add features and instantly improve the experience for everyone.
-
-Check out the [contributing guide](/contributing/overview) to setup your
-development environment and start building.
-
----
+|          |                                                        |                                                                                                                                                             |
+| -------- | ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🏢      | [Project](/design/core-abstractions#project)           | Like a repository on GitHub; this is where you write all your code.                                                                                         |
+| 🪈      | [Pipeline](/design/core-abstractions#pipeline)         | Contains references to all the blocks of code you want to run, charts for visualizing data, and organizes the dependency between each block of code.        |
+| 🧱      | [Block](/design/core-abstractions#block)               | A file with code that can be executed independently or within a pipeline.                                                                                   |
+| 🤓      | [Data product](/design/core-abstractions#data-product) | Every block produces data after it's been executed. These are called data products in Mage.                                                                 |
+| ⏰        | [Trigger](/design/core-abstractions#trigger)           | A set of instructions that determine when or how a pipeline should run.                                                                                     |
+| 🏃‍♂️ | [Run](/design/core-abstractions#run)                   | Stores information about when it was started, its status, when it was completed, any runtime variables used in the execution of the pipeline or block, etc. |
 
 ## 👨‍👩‍👧‍👦 Community
 
-Individually, we’re a mage.
+Individually, we're a mage.
 
 > 🧙 Mage
 >
 > Magic is indistinguishable from advanced technology. A mage is someone who
-> uses magic (aka advanced technology). Together, we’re Magers!
+> uses magic (aka advanced technology). Together, we're Magers!
 
 > 🧙‍♂️🧙 Magers (`/ˈmājər/`)
 >
-> A group of mages who help each other realize their full potential! Let’s hang
-> out and chat together ➝
-
-<Card
-  title="Hang out on Slack"
-  icon="slack"
-  href="https://www.mage.ai/chat"
-></Card>
-
-For real-time news, fun memes, data engineering topics, and more, join us on ➝
-
-<Card
-  title="Twitter"
-  icon="twitter"
-  href="https://twitter.com/mage%5Fai"
-></Card>
-
-<Card
-  title="LinkedIn"
-  icon="linkedin"
-  href="https://www.linkedin.com/company/magetech"
-></Card>
-
-<Card
-  title="GitHub"
-  icon="github"
-  href="https://github.com/mage-ai/mage-ai"
-></Card>
-
----
-
-## 🤔 Frequently Asked Questions (FAQs)
```

Comment:
Are we deprecating the FAQ page?

Reply:
It's a on notion. I think we should move it to mintlify. I should be able to knock that out in about 30m or so, I can make it a follow-up